R/type_boxplot.R

```diff
@@ -94,7 +94,8 @@ draw_boxplot = function(range, width, varwidth, notch, outline, boxwex, staplewe
 
 
 data_boxplot = function() {
-    fun = function(datapoints, bg, col, palette, null_by, null_facet, ...) {
+    fun = function(settings, ...) {
+        list2env(settings, envir = environment())
```

Comment:
A bit worried about inconsistency here compared to, say, `type_abline` where we explicitly extract only the necessary params from the `settings` container, e.g.

```r
list2env(settings[c("datapoints", "bg", "col", "palette", "null_by", "null_facet")], environment())
```

We should decide on one approach and do that for all types.